### DISCUSSION TITLE
Custom keyboard shortcuts

### DISCUSSION THREAD

**Discussion by jankowskimaciej** [grafana/grafana#43172] [Plugins]
I've managed to set up a keyboard shortcut in my custom datasource plugin by adding ` useShortcuts` from `react-shortcuts-hook`.
However, since it's a react hook, it's available only when the `QueryEditor` component is open.
I'd like it to be accessible from the dashboard level as well.

Do you plan to expose `KeybindingSrv` or something similar to register custom combinations?

**Comment by anavss** [comment]
Hi @jankowskimaciej 👋🏼 our product designers are investigating usability improvements to Grafana's keyboard navigation. I spotted your request and wondered if you would be interested in talking to them? You can [apply to participate here](https://ethn.io/32993).

**Reply by jankowskimaciej** [reply]
Hi @anavss, thank you for reaching out. Could you please send the ethn.io link once again? This one has expired.

**Comment by anavss** [comment]
Hi @jankowskimaciej, thanks for your reply. This research study has ended, but if we happen to run a second round, we will reach out to you again 🙂

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
